plugins/calibre.koplugin/metadata.lua

```diff
@@ -42,7 +42,7 @@ end
 
 -- this is the max file size we attempt to decode using json. For larger
 -- files we want to attempt to manually parse the file to avoid OOM errors
-local MAX_JSON_FILESIZE = 30 * 1024 * 1024
+local MAX_JSON_FILESIZE = 30 * 1000 * 1000
```

Comment:
I was quite fine with that one being in mebibytes, but it's not the end of the world, the difference is fairly meaningless in the grand scheme of things ;p.

Reply:
It is to match OP :)